### DISCUSSION TITLE
Add more patterns

### DISCUSSION THREAD

**Discussion by Thelastpoet** [BennyThadikaran/stock-pattern#52] [General]
Are you able to add the following: Rising Wedge, Falling Wedge and Bull Flag?

**Comment by BennyThadikaran** [comment]
The algorithm used in this repo is ideally suited for larger patterns (Since we're using major price pivots). The Wedge and Flags form over a short period like 15 - 20 candles making them a bit tricky.

On the other hand, Harmonic patterns are much more easily implemented. I need to decide which patterns to implement next but flags definitely tops my list.

**Reply by cotton1234** [reply]
Waiting on Flags :)

**Reply by BennyThadikaran** [reply]
I would like to start work on this but there are still bugs that need to be addressed especially on intraday timeframes. I think its best to make sure everything is stable and rock solid, before working on new patterns.

**Reply by cotton1234** [reply]
> I would like to start work on this but there are still bugs that need to be addressed especially on intraday timeframes. I think its best to make sure everything is stable and rock solid, before working on new patterns.

No its better once bugs are fixed in earlier stages. Because when we move to complex stuff, it takes lot of rework to fix the foundation bugs.
